yoshi/tests/run.py

```diff
@@ -0,0 +1,24 @@
+from astropy.table import Table
+
+from yoshi.yoshi import run_one_yoshi
+
+
+def main():
```

Comment:
I still don't know if this is a reasonable place to just stick an example script.

Reply:
The easiest is actually to put it at the top-level because then it just works. It isn't really a test, more an example.

Reply:
Exactly.  I felt weird putting it in the tests directory but didn't know if it had value enough to be set to be configured as an entrypoint script or not.  I've just moved it to top level per this suggestion.